selfservice/errorx/manager.go

```diff
@@ -64,7 +64,8 @@ func (m *Manager) Forward(ctx context.Context, w http.ResponseWriter, r *http.Re
 	to, errCreate := m.Create(ctx, w, r, err)
 	if errCreate != nil {
 		// Everything failed. Resort to standard error output.
-		m.d.Writer().WriteError(w, r, errCreate)
```

Comment:
This is important. Previously, in case of `ctx.Err() == context.Canceled`, we would attempt to write the error container to the database but that would always fail because the context used for that operation is already canceled. We'd then proceed to write back that second error (failing to persist the error container) to the response writer. What we want to do instead is log the persistence error, and output the actual error to the response writer.